src/lib/draw-and-pick.js

```diff
@@ -275,8 +275,14 @@ function getClosestFromPickingBuffer(gl, {
     pickedObjectIndex: -1
   };
 
-  if (deviceX < 0 || deviceY < 0 || deviceX > pickingFBO.width || deviceY > pickingFBO.height) {
-    // x, y out of bounds
+  if (
+    deviceX < 0 ||
+    deviceY < 0 ||
+    deviceX > pickingFBO.width ||
+    deviceY > pickingFBO.height ||
+    layers.length <= 0
```

Comment:
layers.length <= 0 is more of a sanity check.

the getClosestFromPickingBuffer function will be called via layerManager.pickLayer(), thus will be called even if there no layer being added to deck.gl (due to data loading in progress or error).